e2e_test/source/cdc_inline/alter/cdc_table_alter.slt

```diff
@@ -0,0 +1,223 @@
+control substitution on
+
+system ok
+mysql -u root --protocol='tcp' -e "
```

Comment:
```suggestion
mysql -e "
```

Same for others.

Reply:
With mysql launched by RiseDev this should work. 🤔

https://github.com/risingwavelabs/risingwave/blob/87994bfd863f89ab028f9baf0cfde97f694a63e5/e2e_test/commands/mysql#L1-L10